eclipse-jdt/eclipse.jdt.ui
# Add Filter Mechanism for Target in Move Refactoring#18

To let user filter the projects or packages from a search bar in a move refactor dialog.

Fixes https://github.com/vi-eclipse/Eclipse-JDT/issues/18

## What it does
User can now search from the large tree of projects and packages to move the file to. 

## How to test
Go to any file, right click --> Refactor --> Move --> Dialog Box will open with a treeview of Project and Packages --> Type in the search box and see if the filters are working fine 

- [Yes ] I have thoroughly tested my changes
- [ Yes] The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- [ Yes] I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
